README.md

```diff
@@ -140,13 +140,25 @@ In order to use it, you need to create the blacklist table in a migration:
 def change
   create_table :jwt_blacklist do |t|
     t.string :jti, null: false
+    t.datetime :revoked_at, default: Time.now
```

Comment:
I wouldn't add the `revoked_at` column... Usually the clean up will be done just checking the expiration time, it doesn't matter when the token has been revoked. The other clean up strategy I can think is changing the signature key, in which case every blacklist register can be removed.

If someone finds it useful she can rely on default rails timestamps.